sdk/js/src/ConfigRequestConsolidator.ts

```diff
@@ -68,8 +68,8 @@ export class ConfigRequestConsolidator {
                     // don't resolve anything and just make another request while keeping all the previous resolvers
                     this.resolvers.push(...resolvers)
                 } else {
-                    resolvers.forEach(({ resolve }) => resolve(result))
                     this.handleConfigReceivedFunction(result, this.nextUser)
+                    resolvers.forEach(({ resolve }) => resolve(result))
```

Comment:
whats the ordering change here doing again?

Reply:
it was mission critical when I had made the `handleConfigReceivedFunction` async, because it was resolving the promise before all the async operations were complete. Now that were not awaiting the db operations it could go either way. I think it makes more sense to resolve the promise after handling the config being received, but I can also revert it to how it was

Reply:
probably true